tests/src/test/java/alluxio/master/file/CheckConsistencyIntegrationTest.java

```diff
@@ -206,4 +212,60 @@ public void inconsistentFile() throws Exception {
     Assert.assertEquals(expected, mFileSystemMaster
         .checkConsistency(FILE, CheckConsistencyOptions.defaults()));
   }
+
+  /**
+   * Tests the {@link FileSystemMaster#checkConsistency(AlluxioURI, CheckConsistencyOptions)} method
+   * when file or directory mode are changed.
+   */
+  @Test
+  public void chMode() throws Exception {
+    String ufsDirectory = mFileSystem.getStatus(DIRECTORY).getUfsPath();
+    String ufsFile = mFileSystem.getStatus(FILE).getUfsPath();
+    UnderFileSystem ufs = UnderFileSystem.Factory.create(ufsDirectory);
+    ufs.setMode(ufsDirectory, (short) 511);
+    ufs.setMode(ufsFile, (short) 511);
+
+    List<AlluxioURI> expected = Lists.newArrayList(FILE, DIRECTORY);
+    List<AlluxioURI> result =
+        mFileSystemMaster.checkConsistency(new AlluxioURI("/"), CheckConsistencyOptions.defaults());
+    Collections.sort(expected);
+    Collections.sort(result);
+    Assert.assertEquals(expected, result);
+  }
+
+  /**
+   * Tests the {@link FileSystemMaster#checkConsistency(AlluxioURI, CheckConsistencyOptions)} method
+   * when file owner or group are changed.
+   */
+  @Test
+  public void chOwnAndGroupFile() throws Exception {
+    chOwmAndGroup(FILE);
+  }
+
+  /**
+   * Tests the {@link FileSystemMaster#checkConsistency(AlluxioURI, CheckConsistencyOptions)} method
+   * when directory owner or group are changed.
+   */
+  @Test
+  public void chOwnAndGroupDir() throws Exception {
+    chOwmAndGroup(DIRECTORY);
+  }
+
+  private void chOwmAndGroup(AlluxioURI path) throws Exception {
```

Comment:
`chOwm` -> `chOwn`

Do you mind changing the test names from `ch` to `change`?

Reply:
done!